ui/src/main/webapp/src/app/components/project-list/no-projects-welcome.component.html

```diff
@@ -7,7 +7,9 @@
     </div>
 
     <div class="welcome-help-text">
-        To learn more, visit the <a href="https://github.com/windup/windup-web/blob/master/README.md">
-            Red Hat Application Migration Toolkit Web Console Documentation</a>.
+        <template i18n="1st part">To learn more, visit the</template>
```

Comment:
Concrete example of separating unrelated changes into multiple PRs, if it helps explain what people mean by that: each of the two commits in this PR, could just be its own PR (it requires an extra couple clicks to merge both, which is no big deal.)

These two changes are small enough that merging them into one PR is probably fine really, because it doesn't add much headache (I'd submit them separately myself, but I wouldn't gripe if people passed them as one PR.) Where it gets to be confusing is when there are 8 unrelated changes in a PR, or when a couple unrelated changes are tacked into a 500-line PR making it harder to review.

jsight's comment about "1st part" was about something else though -- it was that "1st part" isn't a good identifier for that message. That's a separate issue that's independent of whether it's one PR or two. (Not sure if any of this helps, just trying to be... helpful.)